Resources/About Contents.html

```diff
@@ -1,17 +1,17 @@
 <html>
 <head>
 </head>
-<body style="background-color: #1f1f1f">
+<body style="background-color: BACKGROUNDCOLOR">
 <style type="text/css">
-    h3 {color: #aaaaaa}
-    h5 {text-decoration: plain; color: #aaaaaa}
-    A:link {text-decoration: underline; color: #aaaaaa}
-    A:visited {text-decoration: none; color: #aaaaaa}
-    A:active {text-decoration: none; color: #aaaaaa}
+    h3 {color: TEXTCOLOR}
+    h5 {text-decoration: plain; color: TEXTCOLOR}
+    A:link {text-decoration: underline; color: TEXTCOLOR}
+    A:visited {text-decoration: none; color: TEXTCOLOR}
+    A:active {text-decoration: none; color: TEXTCOLOR}
     body { font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
```

Comment:
Not part of your commits but while I notice this: Likely should use -apple-system font as first preferred font?

Reply:
very true but I won't address this right now